src/prommis/solvent_extraction/sx_model_flowsheet.py

```diff
@@ -7,20 +7,32 @@
 )
 from idaes.core.util.model_statistics import degrees_of_freedom as dof
 
+from idaes.core.util import DiagnosticsToolbox
+from pyomo.util.check_units import assert_units_consistent
+
 from prommis.leaching.leach_solution_properties import LeachSolutionParameters
 from prommis.solvent_extraction.ree_og_distribution import REESolExOgParameters
+from prommis.solvent_extraction.ree_aq_distribution import REESolExAqParameters
 from prommis.solvent_extraction.solvent_extraction import SolventExtraction
 
+import matplotlib.pyplot as plt
```

Comment:
Plotting functions generally should not be part of the unit model file. If necessary, these should be in separate utility modules, and even then there needs to be some demonstrable value (i.e. if it is just using matplotlib to plot a few easy to get trends, then that should be left to the user).

Note that testing plotting functions is also extremely difficult, which is part of why we avoid including it in the model files.

Reply:
Okay I will remove the plots